cmd/stdiscosrv/apisrv.go

```diff
@@ -338,6 +350,14 @@ func certificateBytes(req *http.Request) []byte {
 				}
 			}
 		}
+	} else if hdr := req.Header.Get("X-Tls-Client-Cert-Der-Base64"); hdr != "" {
+		hdr, err := base64.StdEncoding.DecodeString(hdr)
```

Comment:
Maybe add `// Caddy {tls_client_certificate_der_base64}` so we remember why this is here